src/main/resources/application.properties

```
@@ -1,13 +1,35 @@
+# Server Configuration
+server.port=${PORT:8080}
+
+# Datasource Configuration
 spring.datasource.url=jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}
 spring.datasource.username=${DB_USER}
 spring.datasource.password=${DB_PASSWORD}
 spring.jpa.hibernate.ddl-auto=validate
 
+# Azure Cloud Configuration
 spring.cloud.azure.credential.client-id=${AZURE_CLIENT_ID}
 spring.cloud.azure.credential.client-secret=${AZURE_CLIENT_SECRET}
 spring.cloud.azure.profile.tenant-id=${AZURE_TENANT_ID}
 spring.cloud.azure.storage.blob.account-name=${AZURE_STORAGE_ACCOUNT_NAME}
-
 dutact.storage.azure.blob.container-name=${AZURE_STORAGE_BLOB_CONTAINER_NAME}
 
-server.port=${PORT:8080}
\ No newline at end of file
+
+# Token Configuration
+auth.jwt.secret=${JWT_SECRET}
+auth.jwt.lifespan=${JWT_LIFESPAN}
+auth.jwt.algorithm=${JWT_ALGORITHM}
+
+# OTP Configuration
+auth.otp.lifespan=${OTP_LIFESPAN}
```

Actually, I'm too lazy to set env for something like this.
```suggestion
auth.otp.lifespan=${OTP_LIFESPAN:696969} # or something like this
```

According to the following document, it will work, but i'm not sure: https://docs.spring.io/spring-boot/reference/features/external-config.html#features.external-config.files.property-placeholders

If it's feasible to config default value for this, set default value for: **auth.jwt.lifespan**, **auth.jwt.algorithm**, **spring.flyway.locations** (classpath:db/migration for default, when we want seed database, we can include seed folder, maybe)